c/common/src/gballoc.c

```diff
@@ -2,9 +2,11 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 #include <stdlib.h>
+#ifndef WINCE
```

Comment:
I don't think _CRTDBG_MAP_ALLOC is defined by default in CE. If it is I suggest doing a global #undef for it rather than adding WINCE in all source files.